compiler/modules/front.syntax/tests/data/testParser.json

```diff
@@ -390,12 +390,34 @@
     ["V", "stmt", "decimal d = 1E+12d;"],
     ["V", "stmt", "byte i = 10;"],
     ["V", "stmt", "byte[] i = [10];"],
+    ["V", "stmt", "T[2] n = [2, 1.5];"],
```

Comment:
These are all good. But we should have some more tests for things that should *not* be parsed as type descriptors.

```
v[x] = 1;
v[x] += 1;
v[x].foo();
v[2][3].foo();
v[1][n] = bar();
int[x].foo();
int[1] += foo();
```

Reply:
Shall we modify our `testPreparse` for testing `preparseArrayTypeDesc` or just add these as v tests?

Reply:
I don't remember why we need testPreparse. A couple of these cannot be v tests.